### DISCUSSION TITLE
Filer regeneration from raw volume data (.dat, .idx and .vif)

### DISCUSSION THREAD

**Discussion by zguig52** [seaweedfs/seaweedfs#5951] [Q&A] [answered]
Hello,

Is it possible to regenerate filer data directly from raw volume data copy / restoration (?

Just understood that I made my docker filer persistent storage mount wrong... Did not understood why after reboots data was getting lost...

**Comment by chrislusf** [comment, marked as answer]
Not possible. 

You may run https://github.com/seaweedfs/seaweedfs/blob/master/unmaintained/see_dat/see_dat.go to get a list of file names on each volume.

**Reply by zguig52** [reply]
Thanks for your prompt reply. Currently I have 3 filer nodes in a cluster with each one partial data that they can see. Can I do something to resynchronize them otherwhile?

**Reply by chrislusf** [reply]
FYI: Maybe https://seaweedfs.com/ Seaweed Admin the free version can help to have an offsite filer metadata backup.

**Reply by zguig52** [reply]
But no way to resync before modifying the conf?

**Comment by steven2612** [comment]
as far as i understood - if you mount each filer separately and do a `find . -exec touch {} \;` on each filer it should do sync the metadata to the other filers.

**Reply by zguig52** [reply]
This might be a good solution to force resync filers, thanks for the tip. As I did not had so much data on it, what I did to restore coherency is to copy all files from each filer separately, I reinitialized the cluster from scratch and copy back all data. No lost data found yet :)
